itou/www/apply/views/process_views.py

```diff
@@ -415,41 +415,47 @@ def accept(request, job_application_id, template_name="apply/process_accept.html
 
 
 class AcceptHTMXFragmentView(TemplateView):
-    def setup(self, request, job_application_id, *args, **kwargs):
+    def setup(self, request, job_application_id=None, siae_pk=None, *args, **kwargs):
         super().setup(request, *args, **kwargs)
 
-        queryset = JobApplication.objects.siae_member_required(request.user)
-        job_application = get_object_or_404(queryset, id=job_application_id)
-        self.form_accept = AcceptForm(instance=job_application, data=request.POST or None)
+        if siae_pk:
+            siae = Siae.objects.member_required(request.user).get(pk=siae_pk)
```

Comment:
Autrement, 500 lorsque la `siae_pk` n’existe pas (ou que l’utilisateur n’en est pas membre).
```suggestion
            siae = get_object_or_404(Siae.objects.member_required(request.user), pk=siae_pk)
```

Reply:
Évidemment, un test serait bienvenu :)

Reply:
Tests rajoutés (également pour le formulaire sans instance)